airbyte-integrations/connectors/source-close-com/source_close_com/close_com.yaml

```diff
@@ -55,9 +55,10 @@ definitions:
     end_datetime:
       datetime: "{{ now_utc().strftime('%Y-%m-%dT%H:%M:%S.%f%z') }}"
       datetime_format: "%Y-%m-%dT%H:%M:%S.%f%z" # ISO8601
-    step: "365d"
+    step: "P365D"
     cursor_field: "date_updated"
     datetime_format: "%Y-%m-%d"
```

Comment:
Is there a reason why we allow different datetimeformat for start_datetime/end_datetime/datetime_format? Shouldn't we always allow the dateformat of start and end to be as granular as the one used in the API? Here, a user could set up:
* start_datetime: 2021-01-01T05:30:00.000000+0000
* end_datetime: 2021-02-01T12:30:00.000000+0000

... but since the datetime format is "%Y-%m-%d", it would in fact use:

* start_datetime: 2021-01-01
* end_datetime: 2021-02-01

This seems confusing to me since there will be 5 hours 30 minutes more data at the beginning than expected and 11 hours 30 minutes more data at the end than expected.

Reply:
Yup this seems fishy. It also looks like the format of the cursor field is ISO8601 so it would make sense to use that same format for the 3 field (https://developer.close.com/resources/leads/). It would also simplify the component configuration.